RNAndroidPlayground/react-native/build.gradle

```diff
@@ -11,7 +11,7 @@ buildscript {
         }
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:$gradlePluginVersion"
+        classpath 'com.android.tools.build:gradle:3.1.4'
```

Comment:
I would advise to use variable for Gradle version from dependencies.gradle here and also in RNAndroidPlayground/build.gradle.

Reply:
Yeah... Somehow my Android Studio updates those versions, it changes it like this instead of changing the value of the variable.